source/ShopPanel.cpp

```diff
@@ -326,119 +306,6 @@ bool ShopPanel::CanSellMultiple() const
 
 
 
-ShopPanel::TransactionResult ShopPanel::CanDoTransaction(const SDL_Keycode key) const
```

Comment:
CanDoTransaction and DoTransaction were removed and refactored _back_ into KeyDown